lib/google_drive/acl.rb

```diff
@@ -14,6 +14,7 @@ module GoogleDrive
   # This code is based on https://github.com/guyboertje/gdata-spreadsheet-ruby .
   class Acl
     include(Util)
+    include(Enumerable)
```

Comment:
Well, you still need to keep original delegates (`size`, `[]`, `each`). `size` and `[]` are not part of Enumerable. `each` is needed to make Enumerable work.

Reply:
Ah, I already wondered how it would work.

Reply:
Done!